### PR TITLE
Add note regarding api.SpeechSynthesis.pause functionality on Android

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds a note regarding the issues with `SpeechSynthesis.pause()` on Android, as described in #4500.

### CODE REVIEW

File: api/SpeechSynthesis.json [hunk 1]
```diff
@@ -239,7 +239,8 @@
               "version_added": "33"
             },
             "chrome_android": {
-              "version_added": "33"
+              "version_added": "33",
+              "notes": "In Android, <code>pause()</code> ends the current utterance.  As such, <code>pause()</code> behaves just like <code>cancel()</code>."
```
Comment (ddbeck) [inline api/SpeechSynthesis.json]: A little bit of editing here. As a tech writer, I have developed a strong aversion to the word "just", please excuse me

```suggestion
              "notes": "In Android, <code>pause()</code> ends the current utterance. <code>pause()</code> behaves the same as <code>cancel()</code>."
```

File: api/SpeechSynthesis.json [hunk 2]
```diff
@@ -249,7 +250,8 @@
             },
             "firefox_android": [
               {
-                "version_added": "62"
+                "version_added": "62",
+                "notes": "In Android, <code>pause()</code> ends the current utterance.  As such, <code>pause()</code> behaves just like <code>cancel()</code>."
```
Comment (ddbeck) [inline api/SpeechSynthesis.json]:
```suggestion
                "notes": "In Android, <code>pause()</code> ends the current utterance. <code>pause()</code> behaves the same as <code>cancel()</code>."
```

File: api/SpeechSynthesis.json [hunk 3]
```diff
@@ -260,7 +262,8 @@
                     "name": "media.webspeech.synth.enabled",
                     "value_to_set": "true"
                   }
-                ]
+                ],
+                "notes": "In Android, <code>pause()</code> ends the current utterance.  As such, <code>pause()</code> behaves just like <code>cancel()</code>."
```
Comment (ddbeck) [inline api/SpeechSynthesis.json]:
```suggestion
                "notes": "In Android, <code>pause()</code> ends the current utterance. <code>pause()</code> behaves the same as <code>cancel()</code>."
```

File: api/SpeechSynthesis.json [hunk 4]
```diff
@@ -279,7 +282,8 @@
               "version_added": "7.1"
             },
             "webview_android": {
-              "version_added": "4.4.3"
+              "version_added": "4.4.3",
+              "notes": "In Android, <code>pause()</code> ends the current utterance.  As such, <code>pause()</code> behaves just like <code>cancel()</code>."
```
Comment (ddbeck) [inline api/SpeechSynthesis.json]:
```suggestion
              "notes": "In Android, <code>pause()</code> ends the current utterance. <code>pause()</code> behaves the same as <code>cancel()</code>."
```

**Comment by artrayd** [comment]
2021 - Same problem, not working. Also "boundary" event doesn't work.

**Pull request merged**